plugins/tpm-eventlog/fu-plugin-tpm-eventlog.c

```diff
@@ -31,6 +32,30 @@ fu_plugin_destroy (FuPlugin *plugin)
 		g_ptr_array_unref (data->pcr0s);
 }
 
+static gboolean
+fu_plugin_tpm_eventlog_create_dummy (FuPlugin *plugin, const gchar *reason, GError **error)
+{
+	const gchar *key;
+	g_autoptr(FuDevice) dev = fu_device_new ();
+
+	key = fu_plugin_get_dmi_value (plugin, FU_HWIDS_KEY_MANUFACTURER);
+	if (key != NULL)
+		fu_device_set_vendor (dev, key);
```

Comment:
We need a vendor_id too, no?

Reply:
The dummy device from UEFI it's modeled on didn't have one.  If you think so, then yes we should put it in both places.